### ISSUE TITLE
Wording for User's settings for Dart is a bit confusing

### ISSUE THREAD

**Issue by sethladd** [guillermooo/dart-sublime-bundle#271]
Notice the choices for Dart:
- Preferences: Dart Settings - Dart File Type
- Preferences: Dart Settings - Default (Global)

![screen shot 2014-09-11 at 3 52 34 pm](https://cloud.githubusercontent.com/assets/5479/4243102/bf2ac034-3a06-11e4-81b5-3016dee85690.png)

Now, check out what other plugins or system does:

![screen shot 2014-09-11 at 3 56 36 pm](https://cloud.githubusercontent.com/assets/5479/4243115/f07d235c-3a06-11e4-981e-cfb6c60ac0db.png)
- Preferences: Settings - Default
- Preferences: Settings - User

or
- Preferences: Package Control Settings - Default
- Preferences: Package Control Settings - User

In both of those cases, we see two: _Default_ and _User_

Request: change 'Dart File Type' to 'User' ... if that's what it's supposed to be?

Basically, the bug is: I can't find the User's prefs for Dart in either the drop-down menu from the tool bar or when going through the command palette.
